app/controllers/plans_controller.rb

```diff
@@ -246,9 +246,9 @@ def export
     file_name = @plan.title.gsub(/ /, "_")
 
     respond_to do |format|
```

Comment:
I believe respond_to whitelists the formats passed to the block so if we don't want to display html, we can just remove the line format.html...